src/integTest/java/org/sonatype/gradle/plugins/scan/ScanPluginIntegrationTestBase.java

```diff
@@ -373,4 +401,11 @@ private void assertBuildOutputText_OssIndex_Default(BuildResult result, String e
     String resultOutput = result.getOutput();
     assertThat(resultOutput).contains(expectedText);
   }
+
+  private void assertBuildOutputText_ExcludeVulnerabilities_OssIndex(BuildResult result) {
+    String resultOutput = result.getOutput();
+    assertThat(resultOutput).contains("No vulnerabilities found!");
+    assertThat(resultOutput).doesNotContain("commons-collections");
```

Comment:
If not too hard, I'd prefer if the excluded dependency is still listed, but just with 0 vulnerabilities

Reply:
> If not too hard, I'd prefer if the excluded dependency to be still listed, but just with 0 vulnerabilities

nvm, I see that using `showAll` would result in the excluded dependency to be shown :)